source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Domain/Dtos/Validation/ValidationRuleIdentifier.cs

```diff
@@ -41,8 +41,9 @@ public enum ValidationRuleIdentifier
         FeeMustHaveSinglePrice = 25, // VR507-2 / E87
         SubscriptionMustHaveSinglePrice = 26, // VR507-2 / E87
         CommandSenderMustBeAnExistingMarketParticipant = 27, // VR152 / D02
-        UpdateNotYetSupported = 28, // VR902 / D13
+        ChargeUpdateNotYetSupported = 28, // VR902 / D13
         MeteringPointDoesNotExist = 29, // VR200 / E10
         ChargeDoesNotExist = 30, // VR679 / E0I
+        ChargeLinksUpdateNotYetSupported = 31, // VR902 / D13
```

Comment:
nit: I suggest to keep this singular:
```suggestion
        ChargeLinkUpdateNotYetSupported = 31, // VR902 / D13
```